base/uk.ac.stfc.isis.ibex.ui.scriptgenerator/src/uk/ac/stfc/isis/ibex/ui/scriptgenerator/views/ActionsViewTable.java

```diff
@@ -169,7 +169,7 @@ public void setRows(Collection<ScriptGeneratorAction> rows) {
 		if ((!viewer.getTable().isDisposed())) {
```

Comment:
Commenting out this section will restore the original behaviour of the code and give you an idea of what it should look like.
```suggestion
/*
@Override
	public void setRows(Collection<ScriptGeneratorAction> rows) {
		if ((!viewer.getTable().isDisposed())) {
			int focusRow = viewer.getTable().getSelectionIndex();
			int focusColumn = 0;

			
			if (shiftCellFocusToNewlyAddedRow) {
				focusRow = viewer.getTable().getSelectionIndex() + 1;
				shiftCellFocusToNewlyAddedRow = false;
			// if row is not empty
			} else if (focusRow != -1) {
				focusColumn = viewer.getColumnViewerEditor().getFocusCell().getColumnIndex();
			}
			
			viewer.setInput(new WritableList<ScriptGeneratorAction>(rows, null));
			setCellFocus(focusRow, focusColumn);
		}
	} 
	*/
	/**
	 * Sets focus of cell
	 * @param row row number of table
	 * @param column column number of table
	 */
	 /*
	private void setCellFocus(int row, int column) {
		if (row >= 0) {
			viewer.editElement(viewer.getElementAt(row), column);
		}
	}
        */
```